sites/geohub/package.json

```diff
@@ -136,11 +136,16 @@
 		"@azure/web-pubsub": "^1.1.1",
 		"@mapbox/geo-viewport": "^0.5.0",
 		"@mapbox/vector-tile": "^2.0.0",
+		"@maplibre/maplibre-gl-compare": "^0.5.0",
```

Comment:
@lrt-sibat could you remove `@maplibre/maplibre-gl-compare` from dependencies for now?